R/epidatr-package.R

```diff
@@ -1,2 +1,11 @@
 #' @keywords internal
+#' @include cache.R
 "_PACKAGE"
+
+.onLoad <- function(libname, pkgname) {
+  cache_environ$use_cache <<- Sys.getenv("EPIDATR_USE_CACHE", unset = FALSE)
+  cache_environ$use_cache <<- cache_environ$use_cache == "TRUE"
```

Comment:
(no need to use `<<- ` if you are storing in `cache_environ`.  `<<-` is only needed for when you want to replace a variable binding in an ancestor environment rather than form one in your local environment.)

Reply:
yeah, I actually have this fixed locally. Appreciate the catch though